website_product_filters/models/product.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C)2010-  OpenERP SA (<http://openerp.com>). All Rights Reserved
+#    App Author: Vauxoo
+#
+#    Developed by Oscar Alcala
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Lesser General Public License as
+#    published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Lesser General Public License for more details.
+#
+#    You should have received a copy of the GNU Lesser General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    views = fields.Integer(
+        'Views',
+        help='This field shows the nomber of times a product has been\
```

Comment:
In multiline helps it is better:

```python
help="Line 1 "
         "Line 2 "
```

If you use the `\` it will regognize the tabs as blank spaces and will render incorrectly the tooltip.